pollination/annual_daylight/entry.py

```diff
@@ -35,11 +36,10 @@ class AnnualDaylightEntryPoint(DAG):
         alias=north_input
     )
 
-    sensor_count = Inputs.int(
-        default=200,
-        description='The maximum number of grid points per parallel execution.',
-        spec={'type': 'integer', 'minimum': 1},
-        alias=sensor_count_input
+    cpu_count = Inputs.int(
```

Comment:
This is very nice and clear! If we use this naming convention of `cpu_count` across all of the recipes, I think it won't be hard to set a good default for this based on the specs of the machine.